Updated to python3

drkthomp opened the pull request:
I've confirmed results are the same when running python2 and python3 on HCC1395 WGS Exome and RNA Seq Data, though some of them are sorted differently.

Commit:
Updated to python3

adamnovak commented:
Thanks @Dranion!

It looks like @arkal is helping you already, but I wandered in to look at this too.

Just from looking over the PR, it looks like you're adding a `.idea` directory that I don't think wants to be there, and I'm not quite sure the manual changes look like what it would want to say after this gets merged (it still points to your repo, for example).

drkthomp commented:
Thanks for pointing that out! I removed the .idea directory and changed the markdown links to relative references. 

Since I squashed all the commits here, I'm not sure about making changes to the links in changes.md. I could honestly remove it since the results from py2 and py3 were the same, but I don't know what best practice would be.

adamnovak commented:
Changes.md looks good; I hadn't seen that the first time I looked. I think I'm more worried that, bundled with the upgrade, are changes to e.g. README.md that say that this is an experimental Python 3 port, and the original README continues below. If it gets merged, it makes the main project homepage look like this:

![image](https://user-images.githubusercontent.com/5062495/104641006-0f631b80-565e-11eb-87b1-903111298cf4.png)

After this gets merged, it won't be an experimental Python 3 port; it will be the main thing. And the concept of an "original README" won't make sense anymore.

I feel like most of the notes about porting/having ported from Python 2 to Python 3 should probably live in Changes.md under a heading. Then Changes.md would basically be a changelog.

drkthomp commented:
In that case there's a one thing that should probably be addressed for the README to be less experimental, primarily the fact that s3am is python2 only. I have a [quick fork of that](https://github.com/Dranion/s3am) but am just beginning to debug it. Restricting the main version to local only is probably a bad idea in the first place. I'll get that functioning before doing more here.

drkthomp committed:
Merge branch 'BD2KGenomics:master' into 291-update-to-python3
